File: Native/Test/Python/mr-const-array.py
#
#  Copyright 2010-2012 Fabric Technologies Inc. All rights reserved.
#

import fabric

client = fabric.createClient()

ca = client.MR.createConstArray("String", ["one","two","three","four","five"])

count = ca.getCount()
print("ca.getCount() = " + str(count))
for i in range(0, count):
  print("ca.produce("+str(i)+") = " + str(ca.produce(i)))

client.close()

File: Native/Test/Node/mr-const-array.js
FC = require("Fabric").createClient();

ca = FC.MR.createConstArray("String", ["one","two","three","four","five"]);

var count = ca.getCount();
console.log("ca.getCount() = " + count);
for (var i=0; i<count; ++i)
  console.log("ca.produce("+i+") = " + ca.produce(i));

FC.close();
